### DISCUSSION TITLE
Compressed archive in cube example folder?

### DISCUSSION THREAD

**Discussion by cerice1995** [UvaCsl/HemoCell#49] [General discussion]
I am looking for the folder that unpacks into the config files for the cube example, but it doesn't appear to be in the current GitHub version. (Running the bash code "tar -xvf config.tar.xz" as in the cube README results in an error.)

Where can I find the config files for the cube example?

**Comment by gzavo** [comment]
Indeed, this file is missing from the repo. I have attached the missing file as tar.gz here (use "tar -xvzf config.tar.gz").
[config.tar.gz](https://github.com/UvaCsl/HemoCell/files/8665307/config.tar.gz)

**Reply by ivan-pi** [reply]
Here's a hack I learned of recently. If you upload a file through a GitHub comment it gets cached (https://github.com/UvaCsl/HemoCell/files/8665307/config.tar.gz) in what seems to be a stable address. You could now retrieve it using `wget` or `curl` and potentially also checksum it.

**Reply by ivan-pi** [reply]
```txt
$ curl https://github.com/UvaCsl/HemoCell/files/8665307/config.tar.gz > config.tar.gz -s
$ shasum -a 256 config.tar.gz 
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  config.tar.gz
```

**Reply by gzavo** [reply]
Nice trick! Thanks.
